core/nonce.go

```diff
@@ -32,10 +34,14 @@ func NewNonceService() (NonceService, error) {
 		return NonceService{}, err
 	}
 
-	// It is safe to ignore these errors because they only happen
-	// on key size and block size mismatches.
-	c, _ := aes.NewCipher(key)
+	c, err := aes.NewCipher(key)
+	if err != nil {
+		panic("Failure in NewCipher")
```

Comment:
now i'm going to be a jerk and ask you to put the err string into the panic message